styles/layout.css

```diff
@@ -0,0 +1,37 @@
+.hide {
+  display: none;
+}
+
+nav {
+  position: fixed;
+  left: 1em;
+  bottom: 2em;
+}
+
+footer {
+  position: fixed;
+  bottom: 1em;
+  right: 1em;
+  text-align: right;
+}
+
+@media screen and (min-width: 500px) {
```

Comment:
640px is a common CSS break point as this corresponds to landscape width of mobile devices.  There are a few websites that list the common breakpoints (though 640px is a good one to use as a single break point for dev work).